ParisBNF/et/BNFet233.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+   schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BNFet233" type="mss">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title>Poetry in Gəʿəz and Amharic</title>
+            <editor key="JK"/>
+            <editor key="AB" role="generalEditor"/>
+            <funder>Akademie der Wissenschaften in Hamburg</funder>
+         </titleStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                  licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <msDesc xml:id="ms">
+                  <msIdentifier>
+                     <repository ref="INS0303BNF"/>
+                     <collection>Manuscrits orientaux</collection>
+                     <collection>Fonds éthiopien</collection>
+                     <collection>Collection Mondon-Vidailhet</collection>
+                     <idno>BnF Éthiopien 233</idno>
+                     <altIdentifier>
+                        <idno>Mondon-Vidailhet 46</idno>
+                     </altIdentifier>
+                  </msIdentifier>
+               <msContents>
+                  <summary/>
+                  
+                  <msItem xml:id="ms_i1">
+                     <locus from="1r" to="33v"/>
+                     <title type="complete">Amharic poems</title>
```

Comment:
No Lit id?

Reply:
As this is just a stub, I don't think it's necessary to go through the process of creating IDs for all works. Would you agree, @eu-genia and @CarstenHoffmannMarburg ?

Reply:
Yes, I agree. It will be improved in the future.